reference/orders.v2.oas2.yml

```diff
@@ -5415,3 +6157,153 @@ components:
+    orderFees_Resp:
+      title: orderFees_Resp
+      type: object
+      properties:
+        id:
+          description: The unique numeric identifier of the fees object.
+          example: 1
+          type: integer
+        type:
+          description: The type of the fee.
+          type: string
+          enum:
+            - custom_fee
+        display_name_customer:
+          description: The display name of the fee targeting customers.
+          example: Package Protection Insurance
+          type: string
+        display_name_merchant:
+          description: The display name of the fee targeting shoppers.
+            NOTE - At least one of the following fields (display_name_customer, display_name_merchant) must be included in the request.
+          example: Package Protection Fee
+          type: string
+        source:
+          description: TThe source of the request.
+          example: "v2"
+          type: string
+        base_cost:
+          description: |-
+            The base fee cost value. (Float, Float-As-String, Integer)
+          example: 8.5000
+          oneOf:
+            - type: string
+            - type: number
+        cost_ex_tax:
+          description: The fee cost value excluding tax. (Float, Float-As-String, Integer)
+          example: 8.5000
+          oneOf:
+            - type: string
+            - type: number
+        cost_inc_tax:
+          description: The fee cost value including tax. (Float, Float-As-String, Integer)
+          example: 10.0000
+          oneOf:
+            - type: string
+            - type: number
+        cost_tax:
+          description: The tax amount on the fee cost. (Float, Float-As-String, Integer)
+            NOTE - At least two of the following fields (cost_ex_tax, cost_inc_tax, and cost_tax) must be included in the request.
+          example: 1.5000
+          oneOf:
+            - type: string
+            - type: number
+        tax_class_id:
+          description: A unique numeric identifier for the tax class. It defaults to 0 if automatic tax is enabled, the default tax class is applied, or if tax_class_id is absent or null.
+          example: 0
+          type: integer
+    orderFees_Post:
+      title: orderFees_Post
+      type: object
+      properties:
+        type:
+          description: The type of the fee.
+          type: string
+          enum:
+            - custom_fee
+        display_name_customer:
+          description: The display name of the fee targeting customers.
+          example: Package Protection Insurance
+          type: string
+        display_name_merchant:
+          description: The display name of the fee targeting shoppers.
+            NOTE - At least one of the following fields (display_name_customer, display_name_merchant) must be included in the request.
+          example: Package Protection Fee
+          type: string
+        source:
+          description: TThe source of the request.
```

Comment:
Do we allow client to provide this value ? The sample data is "v2" which suggests that this value is auto assigned by our system when fee is created/update by v2 API, is it correct ?

Reply:
The source, yes we allow to pass in, checkout allows too so could be any custom named source I guess

Reply:
thanks for the info Disha, should we provide more realistic sample data for this field, atm it's "v2" and may cause confusion.

Reply:
Ok, using example similar to Checkout now https://github.com/bigcommerce/docs/pull/642/files#diff-2f53894a7f3c1f46d3d98dfd5e4bbb1e2f7a557aa8329ee806a96d047291f6b9R9227